### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by oleomarek**
please approve)

### CODE REVIEW

**Review by helenefialko — APPROVED** [review]
ok)

**Comment by 33alien** [comment]
+

File: l2_vars_and_types/homework/year.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+age = int(input('Enter your age:\n' '\n'))
+print('\n'f'You were born in {2018 - age} year!')
```
Comment (voronov007) [inline l2_vars_and_types/homework/year.py]: use `f` before the whole string
`print(f'\nYou were born in {2018 - age} year!')`

File: l2_vars_and_types/homework/year.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+age = int(input('Enter your age:\n' '\n'))
```
Comment (voronov007) [inline l2_vars_and_types/homework/year.py]: `int(input('Enter your age:\n\n'))`

**Review by voronov007 — APPROVED** [review]
Looks good

**Pull request closed without merging**